test/api/model/model_test.dart

```diff
@@ -212,4 +215,99 @@ void main() {
         .deepEquals([2, 3, 11]);
     });
   });
+
+  group('MessageEditState', () {
+    Map<String, dynamic> baseJson() => deepToJson(eg.streamMessage()) as Map<String, dynamic>;
+
+    test('edit history is absent', () {
+      check(Message.fromJson(baseJson()
+          ..['edit_history'] = null
+      ).editState).equals(MessageEditState.none);
```

Comment:
Can do this with two lines:

```suggestion
      check(Message.fromJson(baseJson()..['edit_history'] = null))
        .editState.equals(MessageEditState.none);
```

This suggestion also includes changing `.editState).equals` to `).editState.equals`, so using the `editState` getter on `Subject<Message>` (thanks to our `MessageChecks` extension) instead of the one on `Message`. We like to do this because it makes the test's output more expressive on failure.